### PR TITLE
..KHADRI-21092024-818-PM-ADDED UPDATE AND DELETE OPERATIONS

### PR DESCRIPTION
**Pull request by SalukulaMahendra**

### CODE REVIEW

**Review by manvithagali52 — APPROVED** [review]

**Review by thasmiyas39 — APPROVED** [review]

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:1]: dont call dao class from this jdbcdatadriver class.

File: src/main/java/com/khadri/jdbc/prepare/data/movie/processer/MovieDataProcesser.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/movie/processer/MovieDataProcesser.java:1]: This you must have dao object and call from here.

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java
Comment (SalukulaMahendra) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:1]: removed MovieDao object in jdbcDataDriver class

File: src/main/java/com/khadri/jdbc/prepare/data/movie/processer/MovieDataProcesser.java
Comment (SalukulaMahendra) [reply src/main/java/com/khadri/jdbc/prepare/data/movie/processer/MovieDataProcesser.java:1]: MovieDao class object called here

**Pull request merged**